server/planning/feed_parsers/superdesk_event_json.py

```diff
@@ -0,0 +1,127 @@
+import logging
+
+from superdesk.io.feed_parsers import FileFeedParser
+from superdesk import get_resource_service
+from superdesk.utc import utcnow
+from planning.common import WORKFLOW_STATE
+import pytz
+import json
+import datetime
+
+utc = pytz.UTC
+logger = logging.getLogger(__name__)
+
+
+class EventJsonFeedParser(FileFeedParser):
+    """Superdesk event specific parser.
+
+    Feed Parser which can parse the Superdesk Event feed and convert to internal event format,
+    but the firstcreated and versioncreated times are localised.
+    """
+
+    NAME = 'json_event'
+
+    label = 'Json Event'
+
+    def can_parse(self, file_path):
+        try:
+            with open(file_path, 'r') as f:
+                superdesk_event = json.load(f)
+                if superdesk_event.get('guid'):
+                    return True
+        except Exception:
+            pass
+        return False
+
+    def parse(self, file_path, provider=None):
+        self.items = []
+        with open(file_path, 'r') as f:
+            superdesk_event = json.load(f)
+        self.items.append(self._transform_from_superdesk_event(superdesk_event))
+        return self.items
+
+    def _transform_from_superdesk_event(self, superdesk_event):
+        ignore_fields = [
+            'files',
+            'state_reason',
+            'schedule_settings',
+            'pubstatus',
+            '_current_version',
+            '_id',
+            'item_id',
+            'actioned_date'
+        ]
+
+        assign_from_local_cv = [
```

Comment:
It might be good to use a `field` -> `vocbulary name` map (as the `_id` differs from the field name).
```python
{
    'anpa_category': 'categories',
    'calendars': 'event_calendars',
    'place': 'locators',
    'occur_status': 'eventoccurstatus'
}
```

`subject` is a special case that is not stored in `vocabularies`, but in a separate service.
```python
from superdesk.io.subjectcodes import get_subjectcodeitems`
```